source/index.rst

```diff
@@ -23,12 +23,24 @@
    api.rst
    terminology.rst
 
+.. Note:: 
+	Adam:
+	I agree with many of Mat's comments on this page. I think that this section might be better off merged with the getting started page and labeled 'Introduction to WEC-Sim'. It can include some examples of WEC-Sims ability and more details to draw users in. What can WEC-Sim simulate? What does a user require to run it? (qualitative overview, not the same technical details and file types in the overview section) Draw users/developers in and introduce them to WEC-sims extensive capabilities
+
+	Other issues:
+	- it doesn't appear on the TOC on the left of the webpage,
+	- it appears as Ch. 17 in the compiled pdf, not at the beginning
+	- In the pdf, the list of authors in the title page runs off the page
+
```

Comment:
Yes, I agree, please work with @H0R5E  to resolve this comment.
- The "index" is the home page, so it shouldn't appear in the index. You can navigate back by clicking WEC-Sim at the top of the page
- As for the PDF.... wouldn't spend too much time resolving it.